xc/common/utils/prjxray_create_place_constraints.py

```diff
@@ -559,6 +559,7 @@ def assign_cmts(self, vpr_grid, blocks, block_locs):
                     )
 
                 if net in self.input_pins:
+                    assert False, "Got a pure input net {}".format(net)
```

Comment:
Not sure if there was an error while committing the changes, because this might just cause the script to fail every time it enters this `if`

Reply:
Yes, that was a mistake... Now the actual code with the fix should be pushed.